tests/integration/landingai/test_predict_e2e.py

```diff
@@ -206,12 +204,12 @@ def test_ocr_predict():
         {
             "text": "公司名称",
             "location": [(99, 19), (366, 19), (366, 75), (99, 75)],
-            "score": 0.828,
+            "score": 0.9974,
```

Comment:
The changes in this test also correspond to an update to the OCR model that was made a while back.